src/styles/global.scss

```diff
@@ -0,0 +1,4 @@
+body {
+  background: var(--color-background-default);
+  color: var(--color-text-default);
+}
```

Comment:
Need to do two extra things: 
1. move our global theme settings out of ThemeProvider.module.scss to global.scss and delete that file. 
2. Remove usages of color: var(--color-text-default); and background: var(--color-background-default); since they're now redundant in most components